pybind11_mkdoc/__init__.py

```diff
@@ -35,9 +35,9 @@ def _append_include_dir(args: list, include_dir: str, verbose: bool = True):
     """
 
     if os.path.isdir(include_dir):
-        args.append(f"-I{shlex.quote(include_dir)}")
+        args.append(f"-I{include_dir}")
```

Comment:
This is correct, unless you are in shell mode, spaces are fine inside strings to subprocess calls.